src/Pos/Util/Tx.hs

```diff
@@ -0,0 +1,142 @@
+-- | Pure functions for operations with transactions
+
+module Pos.Util.Tx
```

Comment:
I don't know what is `Toil` either, though I think I can remember @gromakovsky explaining it to me :trollface: Okay, waiting for him to comment. (Will refrain from renaming the module until then.)